recipes/lzstring/meta.yaml

```diff
@@ -0,0 +1,41 @@
+
+{% set name = "lzstring" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "d54dd5a5f86837ccfc1343cc9f1cb0674d2d6ebd4b49f6408c35104f0a996cb4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - future
+
+  run:
+    - python
+    - future
+
+test:
+  imports:
+    - lzstring
+
+about:
+  home: https://github.com/gkovacs/lz-string-python
```

Comment:
Yup - also, I'm not sure that the upstream version supports all Python versions? That's the main reason that I used this package (also that it's on PyPI as you say).